templates/tfengine/components/devops/terraform.tfvars

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+admins_group = {
+  {{- if not (get .admins_group "exists" false)}}
+  customer_id  = "{{.admins_group.customer_id}}"
+  {{- end -}}
+  {{hclField .admins_group "description" -}}
```

Comment:
I'm assuming `hclFields` are those not required, is that correct?

Reply:
yes! You got it. `hclField` is for optional fields, and foo = {{hcl .foo}} is for mandatory field

Reply:
Great! I'll recheck every field again just to be sure I'm doing it according to the schema